### DISCUSSION TITLE
Error with GLIBC while using emgucv version. - 4.6.0 with Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by parth75** [emgucv/emgucv#864] [Q&A]
**Describe the bug**
We are trying to use emgucv version. - 4.6.0 with Ubuntu 20.04. 
When we use ldd command libcvextern.so to check the missing dependencies we are getting below error - 

./libcvextern.so: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.13' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./libcvextern.so)

./libcvextern.so: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./libcvextern.so)

** OS / Platform **
e.g. Ubuntu 20.04

** .Net version **
e.g.  .Net 6

**CPU Architecture**
e.g. x86_64,

** Emgu CV package used**
NA

**To Reproduce**
NA

**Expected behavior**
NA

**Screenshots**
NA

**Additional context**
NA

**Comment by emgucv** [comment]
If you are using Ubuntu, you should use this nuget package: 
https://www.nuget.org/packages/Emgu.CV.runtime.ubuntu-x64

You can't use the windows runtime on Ubuntu

**Reply by parth75** [reply]
Hello Team, We have used Ubuntu nuget package https://github.com/emgucv/emgucv/tree/4.6.0 which is compatible with Ubuntu 20.04. The link https://www.nuget.org/packages/Emgu.CV.runtime.ubuntu-x64 refers to version 4.7.0 . 
As per github repo for tag https://github.com/emgucv/emgucv/tree/4.7.0 in platforms folders shows support only for Ubuntu 22.04 https://github.com/emgucv/emgucv/tree/4.7.0/platforms/ubuntu/22.04. Could you please help us in resolving the issue for version 4.6.0 which has support for Ubuntu 20.04. Thank you.

**Reply by emgucv** [reply]
For Ubuntu release, we only have resouce to compile for the most current Ubuntu LTS release at the time of the Emgu CV release. For Emgu CV, that means 4.7 is compiled for Ubuntu 22.04; Emgu CV 4.6 is compiled for Ubuntu 20.04. If you need to use Emgu CV 4.7 for Ubuntu 20.04, you will need to compile 4.7.0 for Ubuntu 20.04 yourself. Or, we can provide a custom build for a fee if that is what you need. Please contact support@emgu.com if that is the case. Thank you.

**Reply by parth75** [reply]
Hello Team, above error is expirienced for Emgu CV 4.6 with Ubuntu 20.04 as system OS. Could you please help to get 4.6 emgucv version with Ubuntu 20.04 only. Thanks.
